applications/asset_tracker_v2/src/modules/Kconfig.data_module

```diff
@@ -21,7 +21,7 @@ config DATA_SEND_ALL_DEVICE_CONFIGURATIONS
 
 config DATA_THREAD_STACK_SIZE
 	int "Data module thread stack size"
-	default 5248 if NRF_CLOUD_AGPS
+	default 5632 if NRF_CLOUD_AGPS || LOCATION_METHOD_WIFI
```

Comment:
I had to increase this to avoid crashing in data encoding in `nrf_cloud_codec.c` in this log:
```
	if (IS_ENABLED(CONFIG_CLOUD_CODEC_LOG_LEVEL_DBG)) {
		json_print_obj("Encoded batch message:\n", root_array);
	}
```
This value is good for logging ~30 WiFi access points. I don't know how much we should increase this further and how much.

@simensrostad Comments?

Reply:
Do we really need to encode send 30 access points? If not, I think we should limit this to a reasonable number and possibly avoid increasing the stack size.

Reply:
30 is not even a lot :)
So if we don't send 30, which ones do we leave out. The problem with WiFi is that you have no idea which access points are in the cloud database. And missing the ones that are means that you don't get the location. There is Kconfig option for this `CONFIG_LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT` so Location library won't give you more than that. It's been set to 30 in ATv2 configs.

Reply:
😱 Ok, as long as we make room for the default case.

Reply:
@simensrostad Just as a reminder to ourselves for later: We're reaching quite beefy stack sizes, might want to look into that and see what can be done